sections/appendix/process.tex

```diff
@@ -0,0 +1,36 @@
+\subsection{Project Process}
+\begin{itemize}
+	\item Initial Idea - Combine LDA and PageRank (clustering)
+	\item Preprocessing \& LDA implementation
+	\item Adjacency matrix construction
+	\begin{itemize}
+		\item challenges with calculation time
+		\item focus on more restricting preprocessing and thresholds
+	\end{itemize}
+	\item Bad results
+	\item Discovery of related papers
+	\item Found out our model didn't converge
+\end{itemize}
```

Comment:
ved ikke om vi skal have denne med?

Reply:
Hmm, det er måske dobbeltkonfekt, men den giver alligevel et hurtigt overblik. Der burde nok i hvert fald være en linje efter itemizen der siger f.eks. "This process will now be described further.".